packages/webawesome/src/components/card/card.css

```
@@ -76,10 +76,20 @@
   }
 }
 
+/* Make these slots "full width"  */
+::slotted([slot='header']),
+::slotted([slot='footer']) {
+  flex-grow: 1;
+}
+
+/* Push slots to sides when the action slots renders */
+.has-actions {
+  justify-content: space-between;
+}
```

Any reason not to apply the flex only when actions are slotted?

```suggestion
.has-actions {
  display: flex;
  align-items: center;
  justify-content: space-between;
}
```